CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 3.0.2
+- Add new feature to get_issue to allow for stripping of Jinja templating artifacts.
```

Comment:
It might be useful to give an example of what is meant by "artifacts" here. I can see from the code you mean the curly braces / brackets but one consuming this pack will not know what this means without digging into the source. Just one quick example here would be useful.

Reply:
I added a quiock example